app/models/calendar.rb

```diff
@@ -67,4 +67,12 @@ def content_id
   def body
     @data["body"]
   end
+
+  def christmas?
```

Comment:
I'm being very pedantic, but do we think this should be `christmas_or_boxing_day?`

Reply:
Boxing Day counts as Christmas for me.

But will this even work? Do the Christmas and Boxing Day bank holidays ever get substituted for not 25th/26th?

Reply:
If substituted, I _think_ the most it could be different is two days out because of a weekend - so maybe  `[25, 26, 27, 28]`?

Reply:
Yes, all the bank holidays over Christmas/new year can be substitute days if the actual day falls on a weekend - so 25-28th Dec and 1st-4th Jan can be bank holidays that should have tinsel.

Reply:
Thanks! I pushed two extra commits to your branch that fixes the problem. Feel free to squash them. I didn't want to force-push your branch to do it myself.

Reply:
Happy for you to squash and force-push - don't want to write your commit messages for you @issyl0 !

Reply:
Cool, I'm done now.